src/components/ListView/index.js

```diff
@@ -88,14 +89,14 @@ class ListView extends Component {
       }
     }
 
-    const props = pick(ScrollView.propTypes, this.props);
+    const props = pick(this.props, keys(ScrollView.propTypes));
```

Comment:
Please use `Object.keys` and move the creation of the array outside of `render` since we don't need to recompute it. See https://github.com/necolas/react-native-web/blob/d64df129b2a7f0003e9614549ae70fbcfb734f9e/src/components/TextInput/index.js#L15